assets/js/blocks-registry/payment-methods/test/registry.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { registerPaymentMethodExtensionCallbacks } from '@woocommerce/blocks-registry';
+
+/**
+ * Internal dependencies
+ */
+import { canMakePaymentExtensionsCallbacks } from '../extensions-config';
+
+describe( 'registerPaymentMethodExtensionCallbacks', () => {
+	it( 'Logs an error to console if namespace is already registered', () => {
+		registerPaymentMethodExtensionCallbacks(
```

Comment:
I think it would be good to also test that the namespace is not overwritten. I think this is the main effect we need to test, and the console is secondary. You do something similar for the next test.